### ISSUE TITLE
dataset splitting

### ISSUE THREAD

**Issue by telangdeer** [JeremyCJM/DiffSHEG#29]
Hello, I would like to know the division of the training set, validation set, and test set for the BEAT dataset. I would be very grateful if you could provide a list. Because there is no description of dataset division in CaMN.
